Code Quality/Tools.md

```diff
@@ -6,51 +6,66 @@ If you have already read this section of the Handbook and are here just for the
 
 Git hooks allow us to run scripts during various `git` commands. This verification step can be used to run various tools which ensure that the code satisfies some conditions before it is added to the repository. If the verification fails, the git command will abort.
 
-No matter which code quality tools we use, git hooks are a great way to run those tools. There are multiple ways to add git hooks. For JavaScript projects, we recommend using [husky](https://github.com/typicode/husky). Husky scripts are configured in `package.json`. Here is a basic example which runs `test` before any code is pushed to the repository:
+No matter which code quality tools we use, git hooks are a great way to run those tools. There are multiple ways to add git hooks. For JavaScript projects, we recommend using [husky](https://github.com/typicode/husky).
 
-```js
-// package.json
-{
-  "husky": {
-    "hooks": {
-      "pre-push": "npm test"
-    }
-  }
-}
+Follow [usage](npm set-script prepare "husky install") guidelines for installing Husky and add hooks.
+
+After running all commands described [usage](npm set-script prepare "husky install") chapter, you should have `.husky` folder with hooks folder inside. For example, if you created `pre-commit` hook which runs `npm test` command, you should have `pre-commit` file in `.husky` folder, with following content
+
+```sh
+# .husky/pre-commit
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+npm test
 ```
 
 In this example, if you try pushing and the tests fail, code will not get pushed to the remote. We do not necessarily recommend running tests on push, it is just an example (there are better ways to run automated tests using a proper CI/CD set-up).
 
 Most of our code quality tools are run on either `pre-commit` or `pre-push` hooks, so using git hooks is kind of a prerequisite for the rest of the Code quality handbook section.
 
+Note: By design `husky install` must be run in the same directory as `.git`. You can change directory in your `prepare` script. Also, you will need to change directory in your hooks. For example, if you have `frontend` directory where you want to run `pre-commit` hook, your hook file might look like following
+
+```sh
+# .husky/pre-commit
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+cd frontend && npx lint-staged
+
+```
+
+For more use cases please check [Husky documentation](https://typicode.github.io/husky/#/).
```

Comment:
You could probably link directly to the relevant chapter in the docs: https://typicode.github.io/husky/#/?id=custom-directory

Reply:
Not really, I really wanted to set link for general documentation for all other possible use cases. 😄 
If you have any suggestions for rephrasing this part, please share